pkg/oc/bootstrap/clusterup/componentinstall/privileged.go

```diff
@@ -0,0 +1,36 @@
+package componentinstall
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/openshift/origin/pkg/oc/errors"
+	securityclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+)
+
+func AddPrivilegedUser(clientConfig *rest.Config, namespace, name string) error {
```

Comment:
I think that this should be duplicated to keep clean separation of components. It's a known problem that I don't think will survive another release.